zipline/data/data_portal.py

```diff
@@ -773,19 +773,20 @@ def get_spot_value(self, asset, field, dt, data_frequency):
         if field not in BASE_FIELDS:
             raise KeyError("Invalid column: " + str(field))
 
+        session_label = self.trading_calendar.minute_to_session_label(dt)
+
         if dt < asset.start_date or \
-                (data_frequency == "daily" and dt > asset.end_date) or \
+                (data_frequency == "daily" and
+                    session_label > asset.end_date) or \
```

Comment:
Why do we convert this dt to a label?  Seems like it was using the actual minute before?